include/stdexec/execution.hpp

```diff
@@ -5848,10 +6237,130 @@ namespace stdexec {
             return __self.__env_;
           }
         };
-
     } // __rec
 
-  } // __any
+    namespace __sender {
+      struct __operation_vtable {
+        void (*__start_)(void*) noexcept;
+        void operator()(start_t, void* __op) const noexcept {
+          __start_(__op);
+        }
+      };
+
+      template <class _Op>
+        struct __operation_vtable_fn {
+          constexpr void (*operator()() const noexcept)(void*) noexcept {
+            return +[](void* __object_pointer) noexcept -> void {
+              _Op& __op = *static_cast<_Op*>(__object_pointer);
+              static_assert(operation_state<_Op>);
+              start(__op);
+            };
+          }
+        };
+
+      using __unique_operation_storage = __storage_t<__unique_storage<>, __operation_vtable>;
+
+      template <class _Op>
+        inline constexpr __operation_vtable __op_vtbl{__operation_vtable_fn<_Op>{}()};
+
+      template <class _Op>
+        constexpr const __operation_vtable*
+        tag_invoke(__create_vtable_t<__operation_vtable>, __type<_Op>) noexcept {
+          return &__op_vtbl<_Op>;
+        }
```

Comment:
I try to make all `tag_invoke` overloads hidden friend functions. Hint: the type `_Op` in the signature above is an associated entity of `__type<_Op>` and will be searched for `tag_invoke` overloads.